### PR TITLE
Telemetry: reporting CPU, Memory, Disk and network throughput allocated per task

### PR DESCRIPTION
**Pull request by burdandrei**

### CODE REVIEW

**Comment by dadgar** [comment]
@burdandrei Can you add disk and network

**Comment by burdandrei** [comment]
@dadgar  done!

File: website/source/docs/agent/telemetry.html.md [hunk 1]
```diff
@@ -462,6 +462,30 @@ page for more details.
...
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.disk.allocated`</td>
+    <td>Allocated amount of disk space for this task</td>
+    <td>Bytes</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.client.allocs.<Job>.<TaskGroup>.<AllocID>.<Task>.netwotk.<Device>.mbits.allocated`</td>
```
Comment (dadgar) [inline website/source/docs/agent/telemetry.html.md]: netwotk is mispelled.

**Comment by dadgar** [comment]
@burdandrei Why do you actually want this. This will add a LOT of metrics.

File: website/source/docs/agent/telemetry.html.md (hunk 1 above)
Comment (dadgar) [inline website/source/docs/agent/telemetry.html.md]: Lets instead some up the mbits allocated to the task and ignore the device. (The way it is now, device is not populated)

Comment (burdandrei) [inline website/source/docs/agent/telemetry.html.md]: @dadgar  you wrote that it's a lot of metrics and you are right, I'll try to move this to be emitted only when the allocation is placed, no need to report something that is not changed every 10 seconds

**Comment by multani** [comment]
@burdandrei Do you need any help to move this forward? That would be cool if it could go into 0.6!

**Comment by burdandrei** [comment]
Thanks @multani, @dadgar  made a lot of work on metrics, I'll check the current status and will try to add what was intended =)
In the meantime i hope https://github.com/hashicorp/nomad/pull/2771 will go into 0.6

**Pull request closed without merging**